packages/@ngtools/webpack/package.json

```diff
@@ -25,9 +25,11 @@
     "npm": ">= 3.0.0"
   },
   "dependencies": {
+    "common-tags": "^1.3.1",
```

Comment:
We should not have this dependency, as well as semver, on the `@ngtools/webpack`.